features/events/syslog/pom.xml

```diff
@@ -260,5 +266,30 @@
         </exclusion>
       </exclusions>
     </dependency>
+    
+    <!-- HZN-742 -->
+    <dependency>
+	 <groupId>org.apache.curator</groupId>
+	 <artifactId>curator-test</artifactId>
+	 <version>2.0.0-incubating</version>
+	 <scope>test</scope>
+	</dependency>
+    
+    <dependency>
+	 <groupId>log4j</groupId>
+	 <artifactId>log4j</artifactId>
+	 <version>1.2.15</version>
```

Comment:
Yes. I tried doing it like that. But I get this error
**Missing artifact log4j:log4j:jar:99.99.99-use-log4j2**. But I when mention the version the errors go away.